src/main/java/org/smartregister/domain/Stock.java

```diff
@@ -82,7 +85,7 @@ public Stock(String identifier, String vaccineTypeId, String transactionType, St
 	}
 
 	public Stock(String identifier, String vaccineTypeId, String transactionType, String providerid, int value,
-			StockObjectMetadata stockObjectMetadata, Inventory inventory) {
+			StockObjectMetadata stockObjectMetadata, Inventory inventory, ProductCatalogue product) {
```

Comment:
since product is optional and only passed on 1 endpoint, can we leave it from the constructor

Reply:
Yes, I think we can remove it from here.

Reply:
Done @githengi Please review